alarms.json.default

```diff
@@ -207,5 +207,158 @@
 		"Dragonite":"False",
 		"Mewtwo":"False",
 		"Mew":"False"
+	},
+	"ivs":{
```

Comment:
I want to keep it simple from the exist functions so i just make a new list, very fast solution but sure not good as yours . But changing into the objects like that seem too much works for me, maybe you should handle it after the merge?